inc/Custom_Menu_Items.php

```diff
@@ -261,12 +263,12 @@ public function nav_menu_item_fields( $nav_menu_item_fields, $context ) {
 				?>
 				<p class="field-shortcode description description-wide">
 					<label for="edit-menu-item-shortcode-<?php echo $context['item']->ID; ?>">
-						<?php _e( 'Shortcode', 'custom-menu-item-types' ); ?><br />
+						<?php _e( 'Shortcode Box', 'custom-menu-item-types' ); ?><br />
```

Comment:
```suggestion
						<?php _e( 'Shortcode', 'custom-menu-item-types' ); ?><br />
```
The input is for a shortcode not another shortcode box.